monero-outreach-docs/translations/pt-br/storyboard/getting_started_pt-br.md

```diff
@@ -135,7 +135,7 @@ _Onde Começar?_ [taiga.getmonero.org/project/xmrhaelan-monero-public-relations/
 - Participe de um Grupo de Trabalho - [Ir para Seção](https://www.monerooutreach.org/stories/getting-started-helping-monero.html#workgroups)
 - Contribua com as futuras versões do livro Mastering Monero - [github.com/monerobook/monerobook](https://www.monerooutreach.org/stories/github.com/monerobook/monerobook) & [masteringmonero.com](https://masteringmonero.com/)
 - Responda perguntas no Stack Exchange - [monero.stackexchange.com](https://monero.stackexchange.com/)
-- Faça parte do Marketing de Guerrilla - [monerooutreach.org/guerrilla-toolkit.html](https://www.monerooutreach.org/guerrilla-toolkit.html)
+- Faça parte do Marketing de Guerrilha - [monerooutreach.org/guerrilla-toolkit.html](https://www.monerooutreach.org/guerrilla-toolkit.html)
```

Comment:
Hey @netrik182 take a look at this suggestion:
```suggestion
- Faça parte do Marketing de Guerrilha - [monerooutreach.org/ferramentas-marketing-guerrilha.html](https://www.monerooutreach.org/ferramentas-marketing-guerrilha.html)
```
It would be good that every monerooutreach and getmonero related links have the language link linked.

Reply:
I've seen that the getmonero link have, this is just a reminder.